### ISSUE TITLE
Option --force

### ISSUE THREAD

**Issue by relaxedricky** [hashview/hashview#232]
I wanted to test hashview within a VM just to see what it's like before putting it on other hardware. The only issue is the VM doesn't have a GPU assigned to it.

So when I try to run jobs I get:-

Started: Tue Feb 28 22:24:28 2017
Stopped: Tue Feb 28 22:24:28 2017
job completed
And it only took: 0.15889592899998206 seconds
==== Importing cracked hashes =====
==== import complete ====
==== Crack File Deleted ====
===== creating hash_file =======
===== Hash_File Created ======
===== starting job =======
/usr/local/bin/hashcat -m 500 --potfile-disable --status-timer=15 --outfile-format 3  --outfile control/outfiles/hc_cracked_1_2.txt  -r control/rules/best64.rule control/hashes/hashfile_1_2.txt control/wordlists/password -w 3 | tee -a control/outfiles/hcoutput_1.txt
hashcat (v3.30-382-g50b46d5) starting...

* Device #1: Not a native Intel OpenCL runtime, expect massive speed loss
             You can use --force to override this but do not post error reports if you do so
No devices found/left

Is it possible to set the --force option?

**Comment by i128** [comment]
Agreed.

**Comment by johnnyDEP** [comment]
A temp workaround is to add:
`hcbinpath = hcbinpath + ' --force '`
to /helpers/build_crack_cmd.rb right before the if statements for attack mode

**Comment by recrudesce** [comment]
You can also just add --force to the end of the "Hashcat Binary Path" in the Global Settings.
That's what I do :)

**Comment by i128** [comment]
Issue should now be resolved.

Do a 
`git pull`

To get the latest code, and then run the upgrade instructions found here:

https://github.com/hashview/hashview/wiki/Upgrading-Hashview

**Comment by relaxedricky** [comment]
I don't think ticking this within the webUI is applying the force option. I've had to just ended up changing:

```
    # --force
    if hc_settings.hc_force == true
      cmd = cmd + ' --force'
    end
```
to

```
    # --force
   # if hc_settings.hc_force == true
      cmd = cmd + ' --force'
    #end
```

**Comment by i128** [comment]
found the issue. it was with the web ui not having the correct parameter being passed: force vs hc_force. Fixed and will be merged into v0.7.0

**Comment by ccammilleri** [comment]
0.7 has landed. --force is now supported and working!
